frontends/web/src/api/bitbox02.ts

```diff
@@ -136,13 +136,13 @@ export const restoreFromMnemonic = (
 };
 
 export type TStatus = 'connected'
-  | 'initialized'
-  | 'pairingFailed'
-  | 'require_firmware_upgrade'
-  | 'require_app_upgrade'
-  | 'seeded'
-  | 'unpaired'
-  | 'uninitialized';
+| 'initialized'
```

Comment:
okay this is ugly and seems to be unfixed for years

https://github.com/typescript-eslint/typescript-eslint/issues/1824#issuecomment-1714716151

https://github.com/typescript-eslint/typescript-eslint/issues/455



Seems like there is a debate whether to use formatters or linters for this sort of thing:

https://eslint.style/

https://typescript-eslint.io/linting/troubleshooting/formatting/